app/models/claim/advocate_interim_claim.rb

```diff
@@ -24,27 +24,23 @@ class AdvocateInterimClaim < BaseClaim
       {
         name: :defendants,
         transitions: [
-          {
-            to_stage: :offence_details,
-            condition: ->(claim) { !claim.fixed_fee_case? }
-          },
-          {
-            to_stage: :basic_and_fixed_fees,
```

Comment:
This was not required for advocate interim claims, probably the result of a bad copy/paste 😓 .